test/distributed/org/apache/cassandra/distributed/test/ring/BootstrapTest.java

```diff
@@ -95,7 +96,69 @@ public void autoBootstrapTest() throws Throwable
                 Assert.assertEquals("Node " + e.getKey() + " has incorrect row state", e.getValue().longValue(), 100L);
         }
     }
+    
+    @Test
+    public void hostIdOverrideTest() throws Throwable
+    {
+        int originalNodeCount = 2;
+        int expandedNodeCount = originalNodeCount + 1;
+        try (Cluster cluster = builder().withNodes(originalNodeCount)
+    									.withTokenSupplier(TokenSupplier.evenlyDistributedTokens(expandedNodeCount))
+    									.withNodeIdTopology(NetworkTopology.singleDcNetworkTopology(expandedNodeCount, "dc0", "rack0"))
```

Comment:
@fibersel nit, could you please just do it like "dc1" and "rack1" ? I do not remember I have ever seen "dc0" and "rack0" ever (nor in tests ...)